tests/plugins/test_add_filesystem.py

```diff
@@ -0,0 +1,237 @@
+"""
+Copyright (c) 2016 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+from __future__ import print_function, unicode_literals
+from textwrap import dedent
+from flexmock import flexmock
+
+import re
+import json
+import pytest
+import os.path
+
+import koji
```

Comment:
Think this line needs to be removed. We have to catch import error when importing koji in order to know when we need to use the stub.